rasa/core/tracker_store.py

```diff
@@ -154,16 +157,13 @@ def create_tracker(
 
         Returns:
             The newly created tracker for `sender_id`.
-
         """
-
         tracker = self.init_tracker(sender_id)
 
-        if tracker:
-            if append_action_listen:
-                tracker.update(ActionExecuted(ACTION_LISTEN_NAME))
+        if append_action_listen:
+            tracker.update(ActionExecuted(ACTION_LISTEN_NAME))
 
-            self.save(tracker)
+        self.save(tracker)
```

Comment:
out of the scope of this PR: isn't this redundant? We'd save the tracker anyway later on, wouldn't we?

Reply:
you're right, this fn is only called from `server.py` which always performs a `save()` afterwards